templates/login/login_core.html.twig

```diff
@@ -0,0 +1,267 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    {% if displayGoogleSignin %}
+        <meta name="google-signin-client_id" content="{{ googleSigninClientID }}">
+    {% endif %}
+    <title>{{ title|text }} {{ "Login"|xl }}</title>
+    {% block head %}{% endblock %}
+    <script src="{{ webroot }}/interface/product_registration/product_registration_service.js?v={{ jsIncludes }}"></script>
+    <script src="{{ webroot }}/interface/product_registration/product_registration_controller.js?v={{ jsIncludes }}"></script>
+
+    <script type="text/javascript">
+        document.addEventListener('DOMContentLoaded', () => {
+            document.getElementById('authUser').focus();
+        });
+        $(function () {
+            init();
+
+            var productRegistrationController = new ProductRegistrationController();
+            productRegistrationController.getProductRegistrationStatus(function (err, data) {
+                if (err) { return; }
+
+                if (data.statusAsString === 'UNREGISTERED') {
+                    productRegistrationController.showProductRegistrationModal();
+                }
+            });
+        });
+
+        function transmit_form(element) {
+            // disable submit button to insert a notification of working
+            element.disabled = true;
+            // nothing fancy. mainly for mobile.
+            element.innerHTML = '<i class="fa fa-sync fa-spin"></i> ' + jsText({{ "Authenticating"|xlj }});
+                {% if hasSession %}
+                document.cookie = {{ cookieText|xlj }};
+                {% endif %}
+                document.forms[0].submit();
+            return true;
+        }
+
+        var registrationTranslations = {{ regTranslations }};
+
+        var registrationConstants = {{ regConstants|json_encode }};
+    </script>
+    {{ setupHeader() }}
+</head>
+<body class="login">
+    <form method="POST" id="login_form" autocomplete="off" action="../main/main_screen.php?auth=login&site={{ siteID|attr }}" target="_top" name="login_form">
+        <div class="{{ loginRow|attr }}">
+            <div class="{{ formArea|attr }}">
+                <input type="hidden" name="new_login_session_management" value="1">
+                <input type="hidden" name="languageChoice" value="{{ defaultLangID|attr }}">
+
+                {% if relogin == 1 %}
+                <div class="alert alert-info m-1 font-weight-bold">
+                    {{ "password security has recently been upgraded"|xlt }}. {{ "Please login again"|xlt }}
```

Comment:
lets not change these constants (keep the originals with periods in them) since this is rarely shown and don't want to add 2 more rarely used constants to the translation bloat